plugins/tekton/README.md

```diff
@@ -0,0 +1,39 @@
+# Tekton plugin for Backstage
+
+This plugin will help with visualizing the Pipeline Runs and associated Task Runs powering any service on the Kubernetes cluster.
+
+## Prerequisites
+
+1. Install and configure Tekton plugin by following https://github.com/jquad-group/backstage-jquad#configuration
```

Comment:
Please emphasize that the user should install and configure **only** the backend tekton plugin this way. We want to make sure the user is not confused by this.

Reply:
@tumido user needs to install and configure the complete tekton plugin (both frontend and backend) for this plugin to use the hook directly `usePipelineRunObjects`  https://github.com/janus-idp/backstage-plugins/pull/196/files#diff-da201b26d1d64a686ffa285e470876ed967939b13762860b27a31d745850936aR30.

Otherwise, there would be some duplicity in this frontend plugin.